doc/tutorials/Configure_EDR_service.md

```diff
@@ -43,18 +48,63 @@ Create the following file at the filepath `$ADAGUC_DATASET_DIR/edr.xml`. You can
     </Style>
 
     <!-- Layers -->
+
     <Layer type="database">
-        <FilePath>/data/adaguc-data/HARM_N25_20171215090000_dimx16_dimy16_dimtime49_dimforecastreferencetime1_varairtemperatureat2m.nc</FilePath>
+
+        <FilePath>
+            /data/adaguc-data/HARM_N25_20171215090000_dimx16_dimy16_dimtime49_dimforecastreferencetime1_varairtemperatureat2m.nc</FilePath>
         <Variable units="Celsius">air_temperature__at_2m</Variable>
         <Styles>temperature</Styles>
     </Layer>
-
     <!-- End of configuration /-->
 </Configuration>
 
+```
+
+
+### EdrParameter settings
 
+```xml
+<EdrParameter
+                name="air_temperature__at_2m"
+                unit="°C"
+                standard_name="air_temperature"
+                observed_property_label="Air temperature"
+                parameter_label="Air temperature, 2 metre" />
+```                
+
+- name: Mandatory, Should be one of the WMS Layer names as advertised in the WMS GetCapabilities
+- unit: Mandatory, Sets the unit for the parameter in the parameter_names section of the collection document
+- standard_name: Sets the observedProperty id
+- observed_property_label: Sets the observedProperty label
+- parameter_label: Sets the label for the parameter in the parameter_names section
```

Comment:
If these are not mandatory, where does the data come from?

Reply:
Described this in more detail